packages/main/test/specs/TimePickerClock.spec.js

```diff
@@ -0,0 +1,110 @@
+import { assert } from "chai";
+
+describe("Clock API", () => {
+
+	it("'disabled' property", async () => {
+		await browser.url(`test/pages/TimePickerClock.html`);
+		const hours12 = await browser.$("#myHours12");
+		const hours12Disabled = await browser.$("#myHours12Disabled");
+		const enabled = await hours12.shadow$(".ui5-tp-clock-cover");
+		const disabled = await hours12Disabled.shadow$(".ui5-tp-clock-cover");
+
+		await disabled.click({ x: 45, y: -110, skipRelease: true });
+		await browser.pause(500);
+		assert.strictEqual(await hours12Disabled.getProperty("selectedValue"), 12, "There is no value change after click on 1st number on disabled clock");
+
+		await enabled.click({ x: 45, y: -110, skipRelease: true });
+		await browser.pause(500);
+		assert.strictEqual(await hours12.getProperty("selectedValue"), 1, "There is proper value set after click on 1st number on enabled clock");
+	});
+
+	it("'active' property", async () => {
+		await browser.url(`test/pages/TimePickerClock.html`);
```

Comment:
Don't open the page for every test, this makes the test much more slower.
Create the page as such that you don't need to restart everything.
Use more instances in the HTML page if required.

Reply:
Done